engine/src/main/java/org/terasology/engine/rendering/dag/Node.java

```diff
@@ -72,13 +72,13 @@
      *
      * In the Console, commands have the following syntax:
      *
-     *     dagCommandNode <nodeUri> <command> <arguments...>
+     * {@code dagCommandNode <nodeUri> <command> <arguments...>}
```

Comment:
this would probably also work better inside `<pre>...</pre>`

Reply:
nope. `pre` blocks don't like `<` and `>`